src/pages/theming/dark-mode.md

```diff
@@ -116,6 +116,27 @@ function checkToggle(shouldCheck) {
 <!-- Codepen https://codepen.io/ionic/pen/zYOpQLj -->
 <docs-codepen preview="false" user="ionic" slug="zYOpQLj" height="600px" default-tab="js,result"></docs-codepen>
 
+## Adjusting System UI Components
+
+When developing a dark theme, you may notice that certain system UI components are not adjusting to dark mode properly. To fix this you will need to specify the `color-scheme`. See [color-scheme Browser Compatibility](https://caniuse.com/#feat=mdn-html_elements_meta_name_color-scheme) for details on cross browser support.
+
+While you may be using Ionic components instead of just the native components, `color-scheme` can also affect aspects of your application such as the scrollbar. In order to use `color-scheme` you will need to add the following HTML to the `head` of your application:
```

Comment:
```suggestion
While you may be mainly using Ionic components instead of only native components, `color-scheme` can also affect aspects of your application such as the scrollbar. In order to use `color-scheme` you will need to add the following HTML to the `head` of your application:
```